pages/learn/[slug].tsx

```diff
@@ -42,6 +43,12 @@ const tabs: TabsType[] = [
     slug: "writing-effective-testimony",
     index: 3,
     Component: Write
+  },
+  {
+    label: "The MA Legislative Process",
+    slug: "The MA Legislative Process",
```

Comment:
Let's rename the slug to `legislative-process`

Reply:
But still keep the label was still "The MA Legislative Process" ?

Reply:
Yep!